components/StyledButton/index.js

```diff
@@ -8,4 +8,11 @@ export const StyledButton = styled.button`
   text-decoration: none;
   font-weight: bold;
   border: none;
+  cursor: pointer;
+
+  ${({ $variant }) =>
+    $variant === "delete" &&
+    css`
```

Comment:
Why do you use 'css' here?

Reply:
Without "css" everything is orange in the VS code and we have checked, it was also implemented like this in one of our sample solutions. Is there anything against it?